gulpfile.js

```diff
@@ -308,16 +308,16 @@ function createFaqRedirects() {
     .pipe(gulp.dest(PATHS.dist))
 }
 
-// replaces some values inside json that cant be replaced with handlebars exprerssion since they are inside json
+// replaces some values inside json that cant be replaced with handlebars expression since they are inside json
 function replaceVersionNumbers() {
   return gulp
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.3'))
```

Comment:
```suggestion
    .pipe(replace('[ios.latest-os-version]', '14.4'))
```

Reply:
ok

Reply:
Thanks

Reply:
The changes to version numbers of the OS should be in a separate PR.
This PR is labelled "Update privacy notice on coronawarn.app for version 1.12 [EXPOSUREAPP-4734]" so I wouldn't expect to find other changes like the ones proposed to be piggy-backed onto this PR. It makes it difficult to reconstruct the history of changes and if for some reason the PR for the privacy notices is rejected, then the version changes which are independent will also be rejected.

Reply:
@MikeMcC399 iOS 14.4 is already live on coronwarn.app so I don't know why this PR is outdated in this point.

Reply:
@Ein-Tim 
Sorry for the misunderstanding. I didn't look into all the details of the commits, etc. It just looked strange to be changing the versions by hand. If it is difficult to clean up this PR, then it might be easier to start again!